internal/service/networkpeering/resource_network_peering.go

```diff
@@ -412,11 +414,7 @@ func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 
 	// Updating any of the attributes for Azure Network Peering forces a recreation of the network peering.
 	// Need to check if GCP and AWS have the same behavior
-	switch peer.GetProviderName() {
-	case "GCP":
-		peer.SetGcpProjectId(d.Get("gcp_project_id").(string))
```

Comment:
we're not doing SetGcpProjectId,  SetNetworkName  with GCP now?

Reply:
Yes. If a GCP network peering connection has to be updated, we need to recreate it with the updated values. That is why I put ForceNew in the GCP attributes.

Reply:
ok, got it, thx